packages/query-sync-storage-persister/package.json

```diff
@@ -30,5 +30,8 @@
   "scripts": {
     "clean": "rm -rf ./build",
     "test:eslint": "../../node_modules/.bin/eslint --ext .ts,.tsx ./src"
+  },
+  "dependencies": {
+    "@tanstack/react-query-persist-client": "^4.0.10"
```

Comment:
i think it's wrong that `query-*` packages depend on `react-query-*` packages! It seems that we only import some `types`, so maybe it should just be the other way around and we should in fact move the types ?

Reply:
That is a good point. However the types that are imported are common for both persisters. I would say that we should extract common logic from `react-query-persist-client` to `query-persist-client` so framework-specific adapters would be possible.
But that could be next step.